Topics/Development_Process/Prompt_Engineering.md

```diff
@@ -0,0 +1,97 @@
+
+3. agueness in Instruction: The second prompt is vague in its instructions, missing out on details such as the level of difficulty of the questions or the specific areas of the text to focus on, which could lead to a mismatch in expectations and actual output.
+
+
+## What LLM shoudl you choose?
+
+Here are few popular LLM's you can make use of:
+
+1. LLaMA-2: This model, developed by Meta, is known for its efficiency and scalability, making it suitable for research and development in language understanding and generation. LLaMA-2 is designed to be adaptable to various computational settings, from large-scale servers to smaller, more constrained environments.
+
+2. PaLM-2 (Pathways Language Model): Developed by Google, PaLM-2 is recognized for its advanced capabilities in language understanding, reasoning, and generation. It's particularly adept at handling complex natural language processing tasks, making it suitable for applications requiring deep linguistic analysis and nuanced language generation.
+
+3. GPT (OpenAI): Known for powering ChatGPT, GPT models like GPT-3.5 and GPT-4 are versatile, handling complex reasoning, understanding, and coding tasks. GPT-4, in particular, is multimodal, capable of processing both text and images, making it suitable for a wide range of general-purpose applications​​.
+
+4. BLOOM: An open-access, multilingual language model optimized for text generation and exploring language characteristics. It can generate text in 13 programming languages and 46 natural languages, suitable for multilingual text generation and language studies​​.
+
+5. Claude: Developed by Anthropic, this model is designed for high-reliability text processing tasks including text summarization, creative writing, coding, and search. It offers variations like "Claude Instant" for faster performance​​.
+
+6. Cohere: Created by former Google Brain team members, Cohere offers a range of models for different use cases. Its Cohere Command model is notable for its robustness and accuracy, making it suitable for diverse applications, from content creation to business analytics​​.
+
+7. Falcon: A series of open-source LLMs known for exceptional performance in reasoning and language tasks. The Falcon models, including Falcon 180B, are suitable for applications requiring high-performance language understanding and reasoning​
+
+
+## Additional Resources
+
+1. How to Talk to an LLM: LLM Prompt Engineering for Beginners - UC Today : (https://www.uctoday.com/unified-communications/how-to-talk-to-an-llm-llm-prompt-engineering-for-beginners/#:~:text=,depth%20task%20descriptions)
+
+2. Master Prompt Engineering: Demystifying Prompting Through a Structured Approach : (https://promptengineering.org/master-prompt-engineering-demystifying-prompting-through-a-structured-approach/#:~:text=,can%20quickly%20grasp%20your%20intent) 
+
+3. A guide to prompting llama2 : (https://replicate.com/blog/how-to-prompt-llama) 
```

Comment:
This text is not formatted the same as the ones above. Also the capitalization of LlaMA2